demo/index.html

```diff
@@ -84,25 +88,25 @@
       </template>
     </demo-snippet>
 
-    <h3>To add a leading element, use a paper-icon-item with an <i>item-icon</i> attribute. This
-    leading image can be an iron-icon, or any other regular element.</h3>
+    <h3>To add a leading element, use a paper-icon-item with an <i>item-icon</i> attribute or <code>slot="item-icon"</code>
+    attribute in webcomponents v1. This leading image can be an iron-icon, or any other regular element.</h3>
```

Comment:
Using the `item-icon` attribute doesn't work anymore, right? If so, we should update this comment and remove the attribute from items in the demos.